.github/workflows/main.yml

```
@@ -0,0 +1,36 @@
+---
+name: main
+
+on:
+  pull_request:
+    types: [closed]
+    branches: ['main']
+
+env:
+  GO_VERSION: 1.21.5
+
+jobs:
+  verify:
+    if: github.event.pull_request.merged == true
+    name: Verify
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

We can switch to the v4 version now :)

```suggestion
        uses: actions/checkout@v4
```